calculator.rb

```diff
@@ -0,0 +1,58 @@
+# prints question
+puts "Ready to do some math?"
+# deletes any spaces after the answer provided by the user
+answer = gets.chomp
+
+# ".downcase" takes user input makes it all lowercase
+#  this accounts for any uppercase/lowercase form the word "yes" or "no"
+# depending on user input, if one of the following is true it will prin the line below
+if answer.downcase == "yes"
+  puts "Alrighty then!"
+elsif answer.downcase =="no"
+# "\" escapes quotations, allowing for single quote within the string
+  puts "Whelp, don\'t care!"
+else puts "As if you had an option. HA!"
+end
+
+# creates a visual separation
+puts "--" * 15
+
+puts "Let\'s get some numbers to work with."
+puts "Give me a number: "
+# names each variable input by user and deletes spaces
+first_num = gets.chomp
+puts "And one more number: "
+second_num = gets.chomp
+puts "What operation would you like to perform?"
+operation = gets.chomp
+
+add = ["add", "+", "addition", "plus"]
```

Comment:
Your use of an array is a great way of reducing the crazy amount of booleans you need to deal with in your following conditional block. You should definitely continue use this pattern with subtract, divide, etc.